com.unity.render-pipelines.universal/Shaders/2D/Sprite-Lit-Mask.shader

```diff
@@ -0,0 +1,57 @@
+Shader "Universal Render Pipeline/Sprite-Lit-Mask"
```

Comment:
We should just call it `Universal Render Pipeline/Sprite-Mask`.

Reply:
Changed to Universal Render Pipeline/2D/Sprite-Mask